test/core.vwtest.json

```diff
@@ -5207,7 +5207,44 @@
       "train-sets/dsjson_cb_part1.json",
       "train-sets/dsjson_cb_part2.json"
     ]
+  },
+  {
+    "id": 405,
+    "desc": "test ignore single feature",
+    "vw_command": "--dsjson train-sets/dsjson_cb_part1.json --cb_explore_adf --epsilon 0.2 --invert_hash ignore_feature.interactions --ignore_features_dsjson_experimental FromUrl|timeofday",
+    "diff_files": {
+      "stderr": "train-sets/ref/ignore_feature.stderr",
```

Comment:
do we need stdout/stderr in assert here? I believe the only invariant we are checking is the set of features in the model.
stdout/stderr is just going to be source of noise in every next outputcformat update

Reply:
It does allow us to know if any warnings are produced or error. However, now I say that we should get stdout/stderr if error code is nonzero

Reply:
I've changed my mind to agreeing with alexey :)